src/rest/user_routes.ts

```diff
@@ -1,16 +1,36 @@
-import {Express, Request, Response, Router} from 'express';
+import {Express} from 'express';
 
 import {UserService} from '../services/users/UserService';
-
-
+import {MailService} from '../services/mail/MailService';
 
 export function configureUsersRoutes(app: Express,
-                                     userSvc: UserService): Express {
+                                     userSvc: UserService,
+                                     mailSvc: MailService): Express {
 
   app.get('/users', (req, res) => {
     const users = userSvc.getUsers();
     res.json(users);
   });
 
+  app.post('/users', (req, res) => {
+    const mockUser = {
```

Comment:
You could construct a new Participant with the passed e-mail address and that will "parse" and give you a non-canned name.